README.md

```diff
@@ -10,14 +10,14 @@ Spring Boot Starter support for
 
 ## Usage
 
-Add `spring-boot-starter-handlebars` as dependency:
+Add `handlebars-spring-boot-starter` as dependency:
 ```gradle
 repositories {
     mavenCentral()
 }
 
 dependencies {
-    compile 'pl.allegro.tech.boot:spring-boot-starter-handlebars:0.1.1'
+    compile 'pl.allegro.tech.boot:handlebars-spring-boot-starter:0.1.1'
```

Comment:
0.1.1 -> 0.2.0
Old artefacts (pl.allegro.tech.boot:handlebars-spring-boot-starter:0.1.[01]) should stay in Maven Central.